client/foundries.go

```diff
@@ -506,6 +506,7 @@ func NewApiClient(serverUrl string, config Config, caCertPath string, version st
 		InsecureSkipVerify: config.InsecureSkipVerify,
 	}
 	http.DefaultTransport.(*http.Transport).TLSClientConfig = tlsCfg
+	http.DefaultTransport.(*http.Transport).DisableCompression = true
```

Comment:
Maybe a simple comment like `for CloudFlare` here?
Git blame is not always much helpful for one-line fixes.

Reply:
good call. fixed in force push